example/spec/ingestGranule/IngestUMMGSuccessSpec.js

```diff
@@ -295,8 +298,18 @@ describe('The S3 Ingest Granules workflow configured to ingest UMM-G', () => {
     });
 
     it('publishes CMR metadata online resources with the correct type', () => {
-      const validResources = onlineResources.filter((resource) => resource.Type === 'GET DATA');
-      expect(onlineResources.length).toEqual(validResources.length);
+      const getDataResources = onlineResources.filter((resource) => resource.Type === 'GET DATA');
```

Comment:
We probably want to repeat these tests in `IngestGranuleSuccessSpec.js`

Reply:
yep, this.